website/blog/2024-06-27-new-ferretdb-v122-user-setup-feature.md

```diff
@@ -0,0 +1,76 @@
+---
+slug: new-ferretdb-v122-user-setup-feature
+title: FerretDB releases v1.22 with an initial FerretDB user setup feature
+authors: [alex]
+description: >
+  We have released FerretDB v1.22 with an initial FerretDB user setup feature, configurable document size limit, and several bug fixes and enhancements.
+image: /img/blog/ferretdb-v1.22.0.jpg
+tags: [release]
+---
+
+![FerretDB v1.22](/img/blog/ferretdb-v1.22.0.jpg)
+
+We are thrilled to announce the release of FerretDB v1.22.0, which now includes an initial user setup feature, a configurable document size limit, and several bug fixes and enhancements.
+
+<!--truncate-->
+
+This release, one of the last in the FerretDB v1.x series, adds new features and improvements to make FerretDB even better.
+Users can now set up an initial user for authentication, configure the document size limit, and benefit from several enhancements in this release.
+
+As we continue to build and improve [FerretDB](https://www.ferretdb.com/), we are committed to providing a truly open-source document database that satisfies many MongoDB use cases.
+In the background, we are working on FerretDB v2.0, which will drastically improve performance and compatibility.
+It will also be a complete departure from our current architecture, and we can't wait to share it with you soon.
+
+Read on to learn more about what's new in FerretDB v1.22.0.
+
+## Enable initial FerretDB user setup
+
+A standout feature in FerretDB v1.22 is the ability to set up an initial user for authentication.
+This makes it easier for users to configure their FerretDB instance securely right from the start using dedicated flags or environment variables.
+Here's how you can do it:
+
+```sh
+ferretdb --test-enable-new-auth=true --setup-timeout=1m --setup-username=user --setup-password=pass --setup-database=ferretdb
+```
+
+- `--setup-username`/`FERRETDB_SETUP_USERNAME`: Specifies the username to be created.
+- `--setup-password`/`FERRETDB_SETUP_PASSWORD`: Specifies the password for the user (can be empty).
+- `--setup-timeout`/`FERRETDB_SETUP_TIMEOUT`: Sets the timeout for establishing a working backend connection.
+- `--setup-database`/`FERRETDB_SETUP_DATABASE`: Specifies the database to create the user in.
```

Comment:
Let's say initial database that will be created.
(because that user is not created _in_ this database, but _in_ the `admin` database _for_ that setup)